### PR TITLE
[Mobile Payments] Remove stripe IPP from experiments; always true now

### LINKED ISSUE #6204: [Mobile Payments] Remove IPP with Stripe extension from Experimental Features

### PR DESCRIPTION
**Pull request by allendav**
Closes: #6204

@koke @joshheald - just one review needed but either of y'all are welcome

fyi @malinajirka 

### Description
- Removes Stripe IPP from experiments
- Removes feature flag for Stripe IPP
- `rake generate` was required to update copiables
- Updates `CardPresentPaymentsConfiguration` to just treat it always as enabled/true
- Carefully merges the related changes on trunk for https://github.com/woocommerce/woocommerce-ios/pull/6195

### Testing instructions
- Ensure unit tests pass
- Ensure you can access IPP flows on a store powered by the Stripe extension

### Screenshots
- Experiments looks like this now:

IMG_0314

---
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: Networking/Networking/Model/Copiable/Models+Copiable.generated.swift [hunk 1]
```diff
@@ -1741,8 +1741,8 @@ extension WCPayCardPresentPaymentDetails {
 extension WCPayCardPresentReceiptDetails {
     public func copy(
         accountType: CopiableProp<WCPayCardFunding> = .copy,
-        applicationPreferredName: CopiableProp<String> = .copy,
-        dedicatedFileName: CopiableProp<String> = .copy
+        applicationPreferredName: NullableCopiableProp<String> = .copy,
```
Comment (allendav) [inline Networking/Networking/Model/Copiable/Models+Copiable.generated.swift]: These changes, introduced by `rake generate`, were unexpected. I imagine we forgot to execute `rake generate` on a previous PR.

File: Yosemite/Yosemite/Stores/CardPresentPaymentStore.swift [hunk 2]
```diff
@@ -365,11 +356,6 @@ private extension CardPresentPaymentStore {
     /// Sets the store to use a given payment gateway
     ///
     func use(paymentGatewayAccount: PaymentGatewayAccount) {
-        guard allowStripeIPP else {
-            DDLogError("useStripeAsBackend called when stripeExtensionInPersonPayments disabled")
```
Comment (allendav) [inline Yosemite/Yosemite/Stores/CardPresentPaymentStore.swift:369]: Noteworthy - this would have actually been a bogus error

**Review by koke — APPROVED** [review]
Tests fine 🚀 I took the liberty to push b1ebb3123 before testing to fix a conflict from #6216, which was just merged

**Comment by allendav** [comment]
@koke wrote:

> I took the liberty to push...

Thank you!

**Pull request merged**